airbyte-integrations/connectors/source-omnisend/source_omnisend/omnisend.yaml

```diff
@@ -0,0 +1,83 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["{{ options['name'] }}"]
+  requester:
+    url_base: "https://api.omnisend.com/v3"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: X-API-KEY
+      api_token: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      url_base: "*ref(definitions.requester.url_base)"
+      page_size_option:
+        inject_into: "request_parameter"
+        field_name: "limit"
+      pagination_strategy:
+        type: CursorPagination
+        cursor_value: "{{response['paging']['next']}}"
+        page_size: 100
+      page_token_option:
+        inject_into: "path"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  # API Docs: https://api-docs.omnisend.com/reference/list-contacts
+  contacts_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "contacts"
+      primary_key: "contactID"
+      path: "/contacts"
+  # API Docs: https://api-docs.omnisend.com/reference/get_campaigns
+  campaigns_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "campaigns"
+      primary_key: "campaignID"
+      path: "/campaigns"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      record_selector:
+        extractor:
+          field_pointer: ["campaign"]
```

Comment:
Did you double check this? It is really `campaign`?

Reply:
Yes, I did double-check, it is `campaign` 100%